driver/libetherdream/etherdream.c

```diff
@@ -699,6 +699,14 @@ unsigned long etherdream_get_id(struct etherdream *d) {
 	return d->dac_id;
 }
 
+/* etherdream_get_in_addr(d)
+ *
+ * Documented in etherdream.h.
+ */
+struct in_addr *etherdream_get_in_addr(struct etherdream *d) {
```

Comment:
One request: can you change this (and in the .h file as well) to return a `const struct in_addr *`?